raiden/network/transport/matrix/utils.py

```diff
@@ -260,7 +260,6 @@ def track_address_presence(self, address: Address, user_ids: Set[str]) -> None:
         for uid in user_ids:
             presence = self._fetch_user_presence(uid)
             userids_to_presence[uid] = presence
-            self._set_user_presence(uid, presence)
```

Comment:
Why did we have this before? It should not have done anything because the presence is set above and `_set_user_presence` didn't do anything if the presence was the same.